build/configs/artik053/cxxtest/defconfig

```diff
@@ -282,6 +282,7 @@ CONFIG_PTHREAD_MUTEX_TYPES=y
 CONFIG_PTHREAD_MUTEX_UNSAFE=y
 # CONFIG_PTHREAD_MUTEX_BOTH is not set
 CONFIG_NPTHREAD_KEYS=4
+CONFIG_NPTHREAD_DESTRUCTOR_ITERATIONS=4
```

Comment:
Should we create a proper dependency in the Kconfig entry ? like if  user selects CONFIG_NPTHREAD_KEYS, it should select CONFIG_NPTHREAD_DESTRUCTOR_ITERATIONS ?

Reply:
Already there it is.
```
config NPTHREAD_DESTRUCTOR_ITERATIONS
	int "Maximum number of calling pthread destructor"
	default 4
	depends on NPTHREAD_KEYS != 0
```